substrate/frame/support/procedural/src/lib.rs

```diff
@@ -1142,6 +1142,33 @@ pub fn call_index(_: TokenStream, _: TokenStream) -> TokenStream {
 	pallet_macro_stub()
 }
 
+/// Each dispatchable may also be annotated with the `#[pallet::feeless_if($closure)]` attribute,
+/// which explicitly defines the condition for the dispatchable function to be feeless.
+///
+/// The first argument for the closure must be `account_id: &AccountIdFor<T>`, while the rest
+/// should be the references to arguments of the dispatchable function.
```

Comment:
Please also explain that it only works for signed dispatchables and that the signed extension must be present in the runtime.